matter/si91x/siwx917/BRD4338A/autogen/sl_component_catalog.h

```diff
@@ -24,10 +24,10 @@
 #define SL_CATALOG_MEMORY_MANAGER_PRESENT
 #define SL_CATALOG_NVM3_PRESENT
 #define SL_CATALOG_SEGGER_RTT_PRESENT
-#ifdef DISPLAY_ENABLED
+#if defined(DISPLAY_ENABLED) || defined(SL_ICD_ENABLED)
 #define SL_CATALOG_SLEEPTIMER_PRESENT
 #define SL_CATALOG_DMD_MEMLCD_PRESENT

```

Comment:
The sleeptimer, I can understand, based on your response, that it is needed for both display or ICD. But should
`SL_CATALOG_DMD_MEMLCD_PRESENT` and `SL_CATALOG_GLIB_PRESENT` stay in a separate `ifdef` `DISPLAY_ENABLED` only of did you really mean to enable the MEMLCD/GLIB if `icd` is enabled

Reply:
Ohh true, will move it.